rocAL/rocAL/source/api/rocal_api_augmentation.cpp

```diff
@@ -61,7 +61,7 @@ rocalSequenceRearrange(RocalContext p_context,
     }
     return output;
 }
-
+/*
```

Comment:
why are these commented?

Reply:
The API that are commented out do not have the tensor changes.
We will be issuing separate PR's for these augmentations and API after tensor changes are added.